test/src/main/java/org/axonframework/test/saga/SagaTestFixture.java

```diff
@@ -412,6 +411,9 @@ public FixtureConfiguration registerStartRecordingCallback(Runnable onStartRecor
     public FixtureConfiguration registerListenerInvocationErrorHandler(
             ListenerInvocationErrorHandler listenerInvocationErrorHandler) {
         this.listenerInvocationErrorHandler = listenerInvocationErrorHandler;
+        if (listenerInvocationErrorHandler instanceof RecordingLoggingErrorHandler) {
```

Comment:
This is a bit of an odd measurement to keep backwards compatibility. I was thinking of changing this, so that we could always set it, but throw an `UnsupportedOperationException` when calling `expectSuccessfulHandlerExecution` instead if the errorHandler does not support recording the excception.